roles/ldap_kerberos/tasks/sssd.yml

```diff
@@ -43,10 +50,11 @@
       shell: authselect current
       register: authselect_current
       changed_when: no
+      failed_when: no
```

Comment:
Disable error check is bad. If there is a real error when using `authselect` ansible will continue. Why `authselect current` give error ? Can you investigate the error, maybe find another command or option to fix it instead of ignore error ?

Reply:
`authselect current` gives an error because `authselect` has no active configuration.
```sh
authselect current
# No existing configuration detected.
echo $?
# 2
```
I guess the best thing we can do is ignore the error when return code is `2`, which minimizes the number of cases.

Reply:
Forced push to ignore only `rc == 2`